google-cloud-speech/Rakefile

```diff
@@ -121,6 +121,24 @@ task :jsondoc => :yard do
 
   toc_config = {
     documents: [
+      {
```

Comment:
Do you want `V1p1beta1` to be listed above `V1` when the default API still looks to be `V1`?

Reply:
Hmm, good question. I don't really know what message we're trying to send with the v1p1beta1. Should probably be consistent with your other comment about the `README.md`.

I'll update to prefer V1 unless someone tells us differently.